sherpa-onnx/python/csrc/sherpa-onnx.h

```diff
@@ -5,6 +5,8 @@
 #ifndef SHERPA_ONNX_PYTHON_CSRC_SHERPA_ONNX_H_
 #define SHERPA_ONNX_PYTHON_CSRC_SHERPA_ONNX_H_
 
+#define PYBIND11_DETAILED_ERROR_MESSAGES
```

Comment:
Could you remove it after debugging since it would increase the library file size?

Reply:
yes, I will remove this. Thanks